app/models/project.rb

```diff
@@ -0,0 +1,11 @@
+class Project < ApplicationRecord
+  validates :title, :client_id, presence: true
```

Comment:
we don't need to explicitly check presence of `client_id ` . `belongs_to client`  will do the trick here.